src/api/app/services/worker_status_service/worker_statistics_fetcher.rb

```diff
@@ -0,0 +1,37 @@
+module WorkerStatusService
```

Comment:
:thought_balloon: same here, this is what we introduced the backend library for.